### PR TITLE
styling(forms): Added www modal forms to zoolander

### PR DESCRIPTION
**Pull request by runrog**
Added in modal forms from www

### CODE REVIEW

File: styleguide/_themes/global/scss/forms.scss [hunk 1]
```diff
@@ -114,3 +114,51 @@ table {
...
+.formModal-closeIcon {
+  opacity: .7;
+}
+
+.formModal-title {
+  text-align: center;
+}
+
+.formModal-content {
+  border-radius: 2px;
```
Comment (amberRrucker) [inline styleguide/_themes/global/scss/forms.scss]: we dont have any curved edges anywhere else. I would rather we stay consistent.

File: styleguide/_themes/global/scss/forms.scss [hunk 2]
```diff
@@ -114,3 +114,51 @@ table {
...
+}
+
+.formModal-body {
+  padding: 10px 20px;
+}
+
+.formModal-dialog {
+
+  @media (min-width: $screen-sm-min) {
+    width: 440px;
```
Comment (amberRrucker) [inline styleguide/_themes/global/scss/forms.scss]: let's try not to use fixed widths.

Comment (runrog) [inline styleguide/_themes/global/scss/forms.scss]: @amberRrucker What do you recommend here? All bootstrap modals have fixed widths up until it hits tablet/mobile, at which point it goes fluid. If we go percentage width on desktop, it's either going to look too wide or too small depending on the screen size.

Comment (amberRrucker) [inline styleguide/_themes/global/scss/forms.scss]: ah. I see @runrog. That seems reasonable.

File: styleguide/_themes/global/scss/forms.scss (hunk 1 above)
Comment (runrog) [inline styleguide/_themes/global/scss/forms.scss]: @amberRrucker oops I'll set this to 0.

**Comment by runrog** [comment]
screen shot 2016-08-23 at 1 16 01 pm

**Comment by amberRrucker** [comment]
@runrog Can we do an options for a 50/50 radio button section for like yes/no? And can you set up an example with a real form title? It's really likely they wont ever be that short so it may be helpful to see a longer options. I think it looks a little weird that the x is so far into the form. shouldnt it be more close to the upper right edge there?

**Comment by runrog** [comment]
@amberRrucker 50/50 yes, I'll add that in. I'll add in longer text in the example title, it should just push the form elements down some. The 'x' was the OOB bootstrap close button in modals but I will adjust it, it is a little too far in for my taste also.

**Comment by runrog** [comment]
screen shot 2016-08-23 at 2 14 35 pm

**Comment by amberRrucker** [comment]
+1 looks good to me. @bighappyface or @WidgetsBurritos for code review?

File: styleguide/_themes/global/scss/forms.scss (hunk 2 above)
Comment (WidgetsBurritos) [inline styleguide/_themes/global/scss/forms.scss]: Don't place the media query inside the selector.  Move it outside the selector instead.  

i.e.: 
```
@media (min-width: $screen-sm-min) {
  .formModal-dialog {
    width: 440px;
  }
}
```

Comment (runrog) [inline styleguide/_themes/global/scss/forms.scss]: will fix

**Comment by WidgetsBurritos** [comment]
+1

**Pull request merged**